src/help/faq/README.md

```diff
@@ -54,6 +47,16 @@ In the app, go to Settings > Content Settings.
 
 ---
 
+### I am on the public version of Paperback! When will other manga sources be added?
+Currently other manga sources are already available on the patreon version of the application. Sources on the public version of the app are expected to come most likely at the end of July.
+
+---
+
+### I am on the public version of Paperback! When can I download manga to my device for offline viewing?
+Currently downloading manga is already available on the patreon version of the application. Sources on the public version of the app are expected to come most likely at the end of July.
```

Comment:
Looks like a small mistake. Did you mean "download"?
```suggestion
Currently downloading manga is already available on the patreon version of the application. Manga download on the public version of the app is expected to come most likely at the end of July.
```

Reply:
```suggestion
Currently downloading manga is already available on the patreon version of the application. Source support on the public version of the app is expected to come most likely at the end of July.
```

Reply:
You're right, could have been clarified. What about this instead?

Reply:
Mmh... I believe it is the `When can I download manga` section? Is it really useful to talk about source as there already is `When will other manga sources be added` section above?

But your suggestion could be great for the previous section.

Reply:
I still feel like what I've suggested in the latest edit makes sense for the section! I don't think I quite understand what you're implying on your comment.

Would you mind giving an example as to how you'd suggest modifying things, so I can better understand?

Reply:
I'm not sure we really talk about the same thing 😕 
I agree it totally make sense but I do not understand why do you use the sentence `Source support for the App Store version of the app is expected to come most likely at the end of July.` two times:
* [I am on the public version of Paperback! When will other manga sources be added?](https://deploy-preview-40--paperbackios.netlify.app/help/faq/#i-am-on-the-public-version-of-paperback-when-will-other-manga-sources-be-added)
* [I am on the public version of Paperback! When can I download manga to my device for offline viewing?](https://deploy-preview-40--paperbackios.netlify.app/help/faq/#i-am-on-the-public-version-of-paperback-when-can-i-download-manga-to-my-device-for-offline-viewing)

I may have missed the point of the sentence but I do not understand how *Source support* answer the question "When can I download manga" 🤔

Reply:
Oh! Okay, okay, I understand what you're pointing out now. 
Now that I'm looking through this section again, let me ask another Paperback staff member to look through the FAQ as a whole. Let me get a few opinions on this, and maybe we can get things changed. 

I suggest that we can approve this pull request anyways, since just a small phrasing is an issue still, and take another pass on this later (At least that way, the information is available to the public) - But if you'd prefer to just get this right the first time, I'd understand. Your call.

Let me get back to you after some others take a look!